recipes/mizuroute/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [[ $(uname) == Linux ]]; then
+  ln -s "${CC}" "${BUILD_PREFIX}/bin/gcc"
+  ln -s "${FC}" "${BUILD_PREFIX}/bin/gfortran"
+fi
+
+export F_MASTER="../"
+sed -e "s/FC  =/FC = gnu/g"  \
+    -e "s/FC_EXE =/FC_EXE = gfortran/g" \
+    -e "s/^EXE =/EXE = route_runoff.exe/g" \
+    -e "s/MODE = debug/MODE = fast/g" \
+    -e "s|F_MASTER =|F_MASTER = $F_MASTER|g" \
+    -e "s|NCDF_PATH =|NCDF_PATH=${PREFIX}|g" \
+    route/build/Makefile > route/build/myMakefile
```

Comment:
You should probably just use a patch file

Reply:
Is this a hard requirement? I sort of agree, it might be "cleaner" using `patch` but will end up being just as fragile. Because it won't offer any clear advantage to `sed` for the time being I'd rather not go back and rework it.

I think the better approach will be in a future release of mizuRoute to clean up the makefile itself and then just use environment variables to define these in the build.

Reply:
It is not a hard requirement, but sed is somewhat different on Mac and Linux and you should probably declare it as a build dep if you are going to use it here

Reply:
Also, in my experience, sed is just as fragile as patch. It is worse in some ways, because when sed breaks, sometimes you don't notice until the package has been built and shipped to users, whereas with patch you know it fails during the build

Reply:
Okay, this has been updated for patch.